exporter/model.go

```diff
@@ -243,6 +244,15 @@ func (r *resource) ImportCommand(ic *importContext) string {
 	return fmt.Sprintf(`terraform import %s%s.%s "%s"`, m, r.Resource, r.Name, r.ID)
 }
 
+var (
+	maxRetries = 5
+	retryDelay = 2
+)
+
+func isRetryableError(err string, i int) bool {
```

Comment:
we could have a list of errors on which we would like to retry. Then we can check the list. It would be easier to extend for future use cases otherwise there would be many `||` statements

Reply:
In this case we really need a small number of error retried - the rest should be handled by Go SDK

Reply:
Okay sounds good.

Reply:
I've reimplemented it as I found more error types that we need to handle 🤦